packages/trader/src/AppV2/Containers/Chart/trade-chart.tsx

```diff
@@ -9,18 +9,15 @@ import AccumulatorsChartElements from 'Modules/SmartChart/Components/Markers/acc
 import ToolbarWidgets from 'Modules/SmartChart/Components/toolbar-widgets';
 import useActiveSymbols from 'AppV2/Hooks/useActiveSymbols';
 import useDefaultSymbol from 'AppV2/Hooks/useDefaultSymbol';
-import { isDigitTradeType } from 'Modules/Trading/Helpers/digits';
 
 type TBottomWidgetsParams = {
     digits: number[];
     tick: TickSpotData | null;
 };
-type TBottomWidgetsMobile = TBottomWidgetsParams & {
-    setDigitStats: (digits: number[]) => void;
-    setTickData: (tick: TickSpotData | null) => void;
-};
 
-const BottomWidgetsMobile = ({ digits, tick, setTickData, setDigitStats }: TBottomWidgetsMobile) => {
+const BottomWidgetsMobile = observer(({ digits, tick }: TBottomWidgetsParams) => {
```

Comment:
I was thinking of removing BottomWidgetsMobile into a separate file, but it doesn't even return anything, so I shrug and left it here. Please, let me know if you think it should be separated.